src/main/java/com/google/sps/data/Availability.java

```diff
@@ -0,0 +1,27 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import java.util.LocalDate;
+
+public class Availability {
+  private Person person;
+  private TimeRange when;
+  private LocalDate date;
+  
+  public Availability(Person person, TimeRange when, LocalDate date) {
+    this.person = person;
```

Comment:
Might be better for this to be a Person key,  not a whole person.

Reply:
Done